osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -85,6 +85,14 @@ private void load(OsuColour colours, AudioManager audio, BeatmapManager beatmaps
             }
         }
 
+        protected override void ExitFromBack()
+        {
+            if (modSelect.State == Visibility.Visible)
+                modSelect.Hide();
+
+            base.ExitFromBack();
```

Comment:
This is going to exit the screen anyway, rendering the modselect hiding unnecessary. You likely intended to early return above?

Reply:
oops.